### PR TITLE
Use ribasim.org domain name

### PR DESCRIPTION
**Pull request by visr**
https://ribasim.org/
https://ribasim.nl/

### CODE REVIEW

File: docs/dev/allocation.qmd [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Allocation"
 ---
-# Overview of allocation implementation
+# Overview of allocation implementation [#sec-allocation-overview]
```
Comment (Jingru923) [inline docs/dev/allocation.qmd:4]: image
After rendering, this section looks like this. It doesn't seem correct.

File: docs/dev/allocation.qmd [hunk 2]
```diff
@@ -17,28 +17,28 @@ flowchart TD
     H --> |Yes| D(End)
 ```
 
-If allocation is used in a model, [Allocation structs](https://deltares.github.io/Ribasim/dev/allocation.html#the-allocation-struct) are created.
+If allocation is used in a model, [Allocation structs](#sec-allocation-struct) are created.
```
Comment (Jingru923) [inline docs/dev/allocation.qmd:20]: When I click on this link, it doesn't redirect me the the correct section. Is it the same behavior on your local machine?

File: docs/dev/allocation.qmd
Comment (Jingru923) [inline docs/dev/allocation.qmd:1]: The section links in this document don't direct to the correct sections. Maybe you can recheck it

File: docs/dev/release.qmd [hunk 3]
```diff
@@ -33,7 +33,7 @@ Don't change the old version numbers in `changelog.qmd`.
 
 ## Update the changelog
 
-The `docs/changelog.qmd` file, hosted on https://deltares.github.io/Ribasim/changelog, records the most important changes for users.
+The `docs/changelog.qmd` file, hosted on https://ribasim.org/changelog, records the most important changes for users.
```
Comment (Jingru923) [inline docs/dev/release.qmd:36]: We can make it a hyperlink instead of text 
image

**Review by Jingru923 — CHANGES_REQUESTED** [review]
Looks good! Can you have a look at these comments?

**Comment by visr** [comment]
Thanks. I forgot that it was curly braces, not square braces, to label sections: `# Overview of allocation implementation {#sec-allocation-overview}`

**Review by Jingru923 — APPROVED** [review]
Can you change the hyperlink of "Collect demand" and "Allocate"?

Other things are good!

File: docs/dev/allocation.qmd [hunk 4]
```diff
@@ -17,28 +17,28 @@ flowchart TD
...
 
 The update of allocation (`update_allocation`) is repeating and spread into three parts:
 
 - Updating the mean flows. The mean flow data is only used only for output, not used by any internal functions.
-- ["Collect demand"](https://deltares.github.io/Ribasim/concept/allocation.html#the-high-level-algorithm). This step initialize and solve the optimization problems that collects the demand from the subnetworks.
-- ["Allocate"](https://deltares.github.io/Ribasim/concept/allocation.html#the-high-level-algorithm). This step solves the optimization problems that allocates the demand. For the main network this step allocates to the subnetworks and demand nodes that are in the main network. For the subnetwork this step allocates to the demand nodes.
+- ["Collect demand"](#sec-allocation-overview). This step initialize and solve the optimization problems that collects the demand from the subnetworks.
+- ["Allocate"](#sec-allocation-overview). This step solves the optimization problems that allocates the demand. For the main network this step allocates to the subnetworks and demand nodes that are in the main network. For the subnetwork this step allocates to the demand nodes.
```
[the remark this answers is not in the document]
Comment (visr) [reply docs/dev/allocation.qmd]: Done, didn't realize this was on a different allocation.qmd

**Pull request merged**